ignite/engine/engine.py

```diff
@@ -210,8 +211,7 @@ class TBPTT_Events(CallableEvents, Enum):
     def _handler_wrapper(handler: Callable, event_name: str, event_filter: Callable) -> Callable:
```

Comment:
Here we could probably remove the event_name argument, but I haven't done this so far to avoid breaking changes